### PR TITLE
feat: create preview service

### PR DESCRIPTION
**Pull request by raisedadead**
To test, `docker compose build` and `docker compose up`

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  logger.warn('Invalid URI request attempted.');
+  res.status(404).render('error.njk', { error: 'Invalid URI Requested' });
+});
+
+const PORT = process.env.PORT || 3000;
+app.listen(PORT, () => {
+  logger.info(`Server is running on port ${PORT}`);
+});
+
+export default app;
```
Comment (ojeytonwilliams) [inline src/server.js:73]:
```suggestion
```
Nit: it doesn't seem necessary to export this.

**Review by ojeytonwilliams — COMMENTED** [review]
It looks good overall (and I don't really speak nunjucks), so I only found a couple of nitpicks:

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    build: .
+    ports:
+      - '3000:3000'
+    environment:
+      - NODE_ENV=production
+      - PORT=3000
+    restart: unless-stopped
+    volumes:
+      - .:/usr/src/app
+      - /usr/src/app/node_modules
```
[the remark this answers is not in the document]
Comment (raisedadead) [reply docker-compose.yml:13]: Yeah - I was planning on cleaning up a few things before merging it in.

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Took a look the other day and everything LGTM. I was trying out a few things to get embeds working, but it will take a little more time.

I think we can get this in as-is, and I can open up another PR with fixes for embeds next week. I'll also see if it's possible to move this service back into the main News repo and use the templates there like we discussed.

**Pull request merged**